render_doc.py

```diff
@@ -844,6 +847,40 @@ def get_date(self, version):
         except KeyError:
             raise KeyError("missing release date of Python %s" % version)
 
+    @staticmethod
+    def is_release_tag(tag):
+        return all(v not in tag and 'v' in  tag for v in ['a', 'b', 'c', 'rc'])
+
+    @staticmethod
+    def format_version(version):
+        if version.count('.') == 1:
+            version += '.0'
+        return version[1:] if version.startswith('v') else version
+
+    def _get_all_tags(self):
```

Comment:
Why not using ``git tag -l`` to list tags and then ``git show TAG`` to retrieve the date?

Reply:
> Why not using `git tag -l` to list tags and then `git show TAG` to retrieve the date?

I'm worried that the workflow will become too long. We first need to know that a new version is released, then we should update the local repo, and finally execute the `make update` command. If you use the API you don't need to know the details of the whole process, just simply execute the command. 🤔 

Of course, using `git tag -l` seems to be a faster and more stable solution to get tag information. 
Do I need to modify the current method of getting tag information?

Reply:
Oh. It seems that the `CommitDates` class has used the commit information from the local repository, I missed it. 
It is better to use `git show TAG` to get date information, I will modify the method to get the date.